packages/@expo/cli/src/utils/prompts.ts

```diff
@@ -32,7 +32,7 @@ export default async function prompt(
   { nonInteractiveHelp, ...options }: PromptOptions = {}
 ) {
   questions = Array.isArray(questions) ? questions : [questions];
-  if (env.CI && questions.length !== 0) {
+  if (!isInteractive() && questions.length !== 0) {
```

Comment:
Good that we are showing the question that was supposed to be asked, before failing 👍